### DISCUSSION TITLE
How do you run an example?

### DISCUSSION THREAD

**Discussion by gnublet** [fastapi-users/fastapi-users#1155] [Q&A] [answered]
I'm new to FastAPI, but when using python3.11 or python3.8

After git cloning and going to the `examples/sqlalchemy` folder

I create a virtualenv using `python3.11 -m venv venv`, activate using `source venv/bin/activate`.

Then I install the required packages using `pip install -r requirements.txt`

Then I tried both `python main.py` and `uvicorn app.app:app --reload`, but I get this error

```
raise sa_exc.ArgumentError(
sqlalchemy.exc.ArgumentError: Type annotation for "User.id" can't be correctly interpreted for Annotated Declarative Table form.  ORM annotations should normally make use of the ``Mapped[]`` generic type, or other ORM-compatible generic type, as a container for the actual type, which indicates the intent that the attribute is mapped. Class variables that are not intended to be mapped by the ORM should use ClassVar[].  To allow Annotated Declarative to disregard legacy annotations which don't use Mapped[] to pass, set "__allow_unmapped__ = True" on the class or a superclass this class. (Background on this error at: https://sqlalche.me/e/20/zlpr)
```

Is this expected? A similar error occurs with the oauth example.

**Comment by frankie567** [comment, marked as answer]
It's probably because of the incompatibility with SQLAlchemy 2.0. I've released a new version of the `fastapi-users-db-sqlalchemy` which forces to install SQLAlchemy 1.4. Try to update your dependencies with:

```sh
pip install -U -r requirements.txt
```

That should do it! If not, make sure you have `fastapi-users-db-sqlalchemy==4.0.5`.

I'll release an update to be compatible with SQLAlchemy 2.0 in the coming days.

**Reply by gnublet** [reply]
Thanks, that worked!
